__tests__/app.test.js

```diff
@@ -15,3 +16,14 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects, each with a slug and a description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toEqual(topicsJson);
```

Comment:
We don't want to assert against the test data directly as sometimes sql will add other keys in that the data wont have e.g. ids and therefore we wont be able to get the tests to pass - in this case it is working because the topics table doesn't have any ids in but it's good to not get into the habit of this!